src/pages/index.js

```diff
@@ -49,30 +131,68 @@ const enableValidation = (settings) => {
 
 enableValidation(settings);
 
-// Section class Instance
+/*-----------------------------------------------------------------*/
+/*             Class Instances                          */
+/*-----------------------------------------------------------------*/
+
+// Section instance
 const cardSection = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: (data) => {
-      cardSection.addItem(createCard(data));
+      cardSection.addItem(createCard(data), "append");
     },
   },
   cardContainer
 );
-cardSection.renderItems();
 
-// UserInfo class instance
-const userInfo = new UserInfo({
-  profileName: "#profile_name",
-  profileJob: "#profile_job",
+// Edit Avatar instance
+const popupWithAvatarForm = new PopupWithForm(
+  "#edit-avatar-modal",
+  (formData) => {
+    savingBtn.textContent = "Saving...";
```

Comment:
**Could be improved**

It’s better to make a `renderLoading` method inside `PopupWithForm`. You should find the button once in the constructor and change the text:  

```js
  constructor({ popupSelector, handleSubmit }) {
    super(popupSelector);
    // fix the initial button text only once in the constructor
    this._submitBtnText = this._submitBtn.textContent
  }
  // add 2 params: isLoading and loadingText with a default text
  renderLoading(isLoading, loadingText='Saving...') {
    if (isLoading) {
      this._submitBtn.textContent = loadingText;
    } else {
	// here we return back the initial text. So, you don’t need to bother yourself about it
      this._submitBtn.textContent = this._submitBtnText;
    }
  }

```